LICENSE.md

```diff
@@ -0,0 +1,25 @@
+The MIT License (MIT)
```

Comment:
Don't think this is necessary. The license file is currently simple enough to be in plain text.